src/Light/2band.jl

```diff
@@ -27,7 +27,7 @@
     end
 
     # the rest of the points
-    @unroll for k in grid.Nz-1:-1:1
+    for k in grid.Nz-1:-1:1
```

Comment:
@navidcy for e.g. this `@unroll`, do we have any idea how much time it would save to have it work? Because here (for example) we could give the function Nz in some way that it knows it at compile time, but the extra obfuscation in the code wouldn't be worth it if its not a great time saving

Reply:
I have no idea but you can try and bench?